activerecord/lib/active_record/connection_adapters/postgresql/schema_dumper.rb

```diff
@@ -91,7 +91,7 @@ def prepare_column_options(column)
               spec = { type: schema_type(column).inspect }.merge!(spec)
             end
 
-            spec[:enum_type] = "\"#{column.sql_type}\"" if column.enum?
+            spec[:enum_type] = column.sql_type.inspect if column.enum?
```

Comment:
This use of `.inspect` mirrors `schema_type(column).inspect` above, for non-enum types.